scripts/auth_init.py

```diff
@@ -86,89 +80,88 @@ def random_app_identifier():
     return rand.randint(1000, 100000)
 
 
-def create_server_app_initial_payload(identifier: int):
-    return {
-        "displayName": f"Azure Search OpenAI Chat Server App {identifier}",
-        "signInAudience": "AzureADMyOrg",
-    }
-
-
-def create_server_app_permission_setup_payload(server_app_id: str):
-    return {
-        "api": {
-            "knownClientApplications": [],
-            "oauth2PermissionScopes": [
-                {
-                    "id": "7b207263-0c4a-4127-a6fe-38ea8c8cd1a7",
-                    "adminConsentDisplayName": "Access Azure Search OpenAI Chat API",
-                    "adminConsentDescription": "Allows the app to access Azure Search OpenAI Chat API as the signed-in user.",
-                    "userConsentDisplayName": "Access Azure Search OpenAI Chat API",
-                    "userConsentDescription": "Allow the app to access Azure Search OpenAI Chat API on your behalf",
-                    "isEnabled": True,
-                    "value": "access_as_user",
-                    "type": "User",
-                }
+def server_app_initial(identifier: int) -> Application:
```

Comment:
Do we even need this method anymore? 1 liners can be inlined IMO

Reply:
No, but I think in this case, it's nice and symmetrical with the other Application objects, so it's somewhat easier to read when it's inside a function like the others, I think. (It reads top to down)